roles/network_plugin/cilium/defaults/main.yml

```diff
@@ -152,6 +152,12 @@ cilium_hubble_install: false
 ### Enable auto generate certs if cilium_hubble_install: true
 cilium_hubble_tls_generate: false
 
+### Capacity of Hubble events buffer. The provided value must be one less than an integer power of two and no larger than 65535
+### (ie: 1, 3, ..., 2047, 4095, ..., 65535) (default 4095)
+# cilium_hubble_event_buffer_capacity: 4095
```

Comment:
Please also update this to https://github.com/kubernetes-sigs/kubespray/blob/master/inventory/sample/group_vars/k8s_cluster/k8s-net-cilium.yml

Reply:
Maybe we could avoid errors/confusion by either:
- validating the value
- using the power of two in the variable and calculating the final value ?

```
cilium_hubble_event_buffer_capacity_power_two: 16 # very bad name but you get the idea
```

Reply:
I think it's clearer to validate the value... typically you can check if an integer is a power of 2 by using a bitwise and.  It looks like jinija doesn't have support for bitwise operations... https://github.com/pallets/jinja/issues/249

I could create an ansible bitwise filter to do this... but I think it's simpler just to check for a range of valid values.

Something like this:
```yaml
...
    - name: Check if cilium_hubble_event_buffer_capacity is a power of 2 minus 1 and less than 65535
      assert:
        that:
          - "cilium_hubble_event_buffer_capacity in [1, 3, 7, 15, 31, 63, 127, 255, 511, 1023, 2047, 4095, 8191, 16383, 32767, 65535]"
        msg: "Error: cilium_hubble_event_buffer_capacity is not a power of 2 minus 1 and it should be between 1 and 65535."
      when: cilium_hubble_event_buffer_capacity is defined
```

Reply:
added those checks here:
https://github.com/kubernetes-sigs/kubespray/compare/ca8ce8d947f66adf07898196d68041a95c2bf9af..6f4aa87a905feb2e3224229fef64406d8fa74864#diff-54b620bf4d912490276c851c3d1bf285f3efce9483973a67eb0e8f80079e1a0cR65